libs/execution_context/ebpf_maps.c

```diff
@@ -183,8 +183,10 @@ _create_array_map(
     ebpf_handle_t inner_map_handle,
     _Outptr_ ebpf_core_map_t** map)
 {
-    if (inner_map_handle != ebpf_handle_invalid)
-        return EBPF_INVALID_ARGUMENT;
+
+    // Temporarily removing check for inner map handle until
```

Comment:
Assuming we expect the old code to be put back in, we should reference it somehow. Currently the
"check for inner map handle" in the commend doesn't make sense without this context.

Reply:
Won't the file history be sufficient to restore the code? I could put back the old code and comment it out. Would you prefer that?

Reply:
In this case, yes because it's temporary and you already have the issue number in the comment.